hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelineManagerMXBean.java

```diff
@@ -36,4 +36,6 @@ public interface PipelineManagerMXBean {
    */
   Map<String, Integer> getPipelineInfo() throws NotLeaderException;
 
+  Map<String, String[]> getPipelineLeaders() throws NotLeaderException;
```

Comment:
`throws NotLeaderException` is redundant

Reply:
Thanks for the find, I have removed it !!